better-code/src/chapter-2-contracts.md

```diff
@@ -479,7 +482,7 @@ neither correct nor incorrect; it does something, but does it do the
 
 > All undocumented software is waste. It's a liability for a company.
 >
-> —Alexander Stepanov (https://youtu.be/COuHLky7E2Q?t=1773)
+> &mdash; Alexander Stepanov (<https://youtu.be/COuHLky7E2Q?t=1773>)
```

Comment:
You added a space after the dash.  I don't think that's how this is done ([non-authoritative citation](https://www.wisdomandwonder.com/writing/13270/what-dash-should-i-use-when-attributing-a-quotation)).

```suggestion
> &mdash;Alexander Stepanov (<https://youtu.be/COuHLky7E2Q?t=1773>)
```